docs/dev/table/tableApi.md

```diff
@@ -1682,6 +1682,34 @@ The `OverWindow` defines a range of rows over which aggregates are computed. `Ov
 
 {% top %}
 
+### Map
```

Comment:
Would it better to put Map together with `Scan, Projection, and Filter`? We can add another Row to the Table.

Reply:
I think Map is a new operator of table, I recommend a separate description, flatmap is the same. What do you think?

Reply:
I personally prefer a separate description as map/flatMap/aggregate/flatAggregate will introduce some new concepts which cannot be mapped directly to common SQL operations and a separate description will make the concept more clear.

Reply:
How about adding a `Row-based operations` contains `map/flatmap/aggregate/flatAggregate`. ?